core/lib/GlobalPool.ts

```diff
@@ -98,7 +98,9 @@ export default class GlobalPool {
   ): Promise<Session> {
     this._activeSessionCount += 1;
     try {
-      return await this.chromeCore.createSession(options);
+      const session = await this.chromeCore.createSession(options);
+      if (!session) this._activeSessionCount -= 1;
+      return session;
```

Comment:
createSession can return no session and yet not throw an error?

Reply:
I wasn't sure what to do here... this will only happen during shutdown on a pending creation.

Reply:
I'm looking at the createSession in ChromeCore... it seems our current setup will have weird side affects... not sure what to do. It seems weird that createSession would return undefined... what happens to the dev's code that is expecting a session back? Should an error be thrown? 

This PR probably isn't the one to fix it.